static/app/components/events/groupEventEntries.spec.tsx

```diff
@@ -293,4 +295,116 @@ describe('GroupEventEntries', function () {
       });
     });
   });
+  describe('Rendering', function () {
+    it('renders the Resources section for Performance Issues', function () {
+      const group: Group = TestStubs.Group();
+      group.issueCategory = IssueCategory.PERFORMANCE;
```

Comment:
we should probably add the ability to pass a group type as an argument to `TestStubs.Group()`

Reply:
I think these stubs are built so you can pass in whatever args you want, so it works when I do this: `TestStubs.Group({issueCategory: IssueCategory.PERFORMANCE});`

Reply:
great!